docs/debug.MD

```diff
@@ -102,7 +102,6 @@ AC Debugger:      at org.apache.commons.io(org.apache.commons.io.filefilter.OrFi
 AC Debugger:      at bundle-0(java.io.File:1291) <instance of java.io.File(id=16236)>
 AC Debugger:      at org.apache.commons.io(org.apache.commons.io.FileUtils:473) <class of org.apache.commons.io.FileUtils>
 AC Debugger:      at org.apache.commons.io(org.apache.commons.io.FileUtils:524) <class of org.apache.commons.io.FileUtils>
```

Comment:
ℹ️ This change is because the logic to analyze the stack was changed to properly account for the frame that comes after a `doPrivileged()` call as it should have been. As such, there is no need to duplicate that frame after the call to `doPrivileged()` block to see its domain as the analysis will be accounted for it.